guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<ul><li>Instead of <em>cross-platform, </em>I think it'd be clearer is it said <em>between ArcGIS Online and ArcGIS Enterprise organizations</em> or as just <em>between organizations.</em></li><li>Change the syntax of the note text to render it offset in the production html page</li></ul><p> </p><p>&gt; _Note:_ This workflow is for demonstration purposes only. To replicate this, you may have to sign-in to an ArcGIS Enterprise organization you have access to.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>